README.md

```diff
@@ -66,9 +66,10 @@ positional arguments:
   receive_port       The port where the osc_qasm.py Server will listen for
                      incoming messages. Default port is 1416
   send_port          The port that osc_qasm.py will use to send messages back
-                     to Max. Default port is 1417
-  ip                 The IP address where the client (Max/MSP) is located.
-                     Default IP is 127.0.0.1 (localhost)
+                     to Max/MSP. Default port is 1417
+  ip                 The IP address to where the retrieved results will be
+                     sent (Where Max/MSP is located). Default IP is 127.0.0.1
+                     (localhost)
```

Comment:
Please bring these updates to [osc_qasm.py](https://github.com/iccmr-quantum/OSC-Qasm/blob/main/osc_qasm.py) as well.